### PR TITLE
Crimson Dust cards

### PR DESCRIPTION
**Pull request by danhut**
Might need to wait until cards are released / images to push to live?

Implement all of Crismon Dust.  Other than Bamboo Dome.

### CODE REVIEW

File: src/clj/game/cards/agendas.clj [hunk 1]
```diff
@@ -165,6 +165,13 @@
     :end-turn {:effect (effect (lose :runner :tag 2))
                :msg "make the Runner lose 2 tags"}}
 
+   "CFC Mining Contract"
```
Comment (JoelCFC25) [inline src/clj/game/cards/agendas.clj]: This card's English title is "CFC Excavation Contract"  ([link]()http://www.cardgamedb.com/index.php/netrunner/android-netrunner-card-spoilers/_/red-sand-cycle/crimson-dust/))

File: src/clj/game/cards/agendas.clj [hunk 2]
```diff
@@ -321,6 +328,13 @@
     :events {:pre-ice-strength {:req (req (has-subtype? target "Code Gate"))
                                 :effect (effect (ice-strength-bonus 1 target))}}}
 
+   "Enhanced AR Security"
```
Comment (JoelCFC25) [inline src/clj/game/cards/agendas.clj]: AR-Enhanced Security

File: src/clj/game/cards/events.clj [hunk 3]
```diff
@@ -1164,6 +1164,13 @@
    "Recon"
    {:prompt "Choose a server" :choices (req runnable-servers) :effect (effect (run target nil card))}
 
+   "Reformation"
```
Comment (JoelCFC25) [inline src/clj/game/cards/events.clj]: Reshape

File: src/clj/game/cards/operations.clj [hunk 4]
```diff
@@ -1003,6 +1018,26 @@
       :effect (req (move state side target :hand)
                    (resolve-ability state side (replant 1) card nil))})
 
+   "Restoration"
```
Comment (JoelCFC25) [inline src/clj/game/cards/operations.clj]: Restore

File: src/clj/game/cards/operations.clj [hunk 5]
```diff
@@ -818,6 +818,13 @@
                                     (+ c (count (filter (fn [ice] (:rezzed ice)) (:ices server)))))
                                   0 (flatten (seq (:servers corp))))))}
 
+   "Planned Power Outage"
```
Comment (JoelCFC25) [inline src/clj/game/cards/operations.clj]: Rolling Brownout

File: src/clj/game/cards/resources.clj [hunk 6]
```diff
@@ -323,6 +328,9 @@
                    :msg "gain 1 [Credits]"
                    :effect (effect (gain :runner :credit 1))}}}
 
+   "Corporate Defectors"
```
Comment (JoelCFC25) [inline src/clj/game/cards/resources.clj]: Defector, singular

**Comment by danhut** [comment]
Thanks Joel!  You can tell where I am working from ;)

**Comment by PesiTheWizard** [comment]
Re:Breached Dome update

Won't this result in two messages per access?

**Comment by JoelCFC25** [comment]
Yes. The `:msg` could be dropped from Breached Dome.

**Comment by danhut** [comment]
Well I think I fixed all the cards I did before I had any ability to test them in the engine!  Good to go!

**Comment by mtgred** [comment]
👍

**Pull request merged**